osquery/utils/conversions/windows/strings.cpp

```diff
@@ -15,9 +15,34 @@
 namespace osquery {
 
 // Helper object used by Wide/Narrow converter functions
-static std::wstring_convert<
-    std::codecvt_utf8_utf16<wchar_t, 0x10ffff, std::little_endian>>
-    converter;
+
+struct utf_converter {
+  std::wstring from_bytes(const std::string& str) {
+    std::wstring result;
+    if (str.length() > 0) {
+      result.resize(str.length() * 2);
+      int count = MultiByteToWideChar(
```

Comment:
nit: I think we'd tend to prefer leveraging `auto count = ...` here, but no biggy.

Reply:
Agree. Let me think about this PR a bit more.... I might cancel it.

Reply:
Ok, found the issue why it was displaying `fr°tz-jorn` instead of `frøtz-jorn`. We needed to set the console's code page to UTF-8. I'll update the original issue with more information.